incubator/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -141,6 +141,8 @@ spec:
       - name: data
         emptyDir: {}
   {{- else }}
```

Comment:
this depends on persistence being enabled - should it not be independent of that (even if it makes most sense with a "proper" setup with persistence?

Reply:
@davidkarlsen 
You're right. Fixed.